CONTRIBUTING.md

```diff
@@ -2,156 +2,251 @@
 
 ## Running and Developing DB
 
+### Introduction 
+
+Welcome to the Platformatic development guide. This document will help you set up your development environment, run the [Platformatic DB](https://docs.platformatic.dev/docs/db/overview), and contribute to the project. Follow the steps below to get started.
+
 ### Preparation
 
-1. Clone this repository, and enter in the directory (`cd platformatic` on most installations)
-2. Install pnpm `npm i pnpm --location=global` (if you're on Mac, you can use `brew` to install it with `brew install pnpm`)
-3. Make sure to run `pnpm setup` to configure the local path; then close and reopen your terminal.
-4. Install dependencies `pnpm i`
-5. Install docker with Docker Desktop or [Colima](https://github.com/abiosoft/colima)
+1. **Clone the repository**: Clone this repository and navigate to the platformatic directory.
+
+```sh
+git clone https://github.com/platformatic/platformatic.git
+cd platformatic
+```
+
+2. **Install pnpm**: Install pnpm: Follow the official [pnpm installation guide](https://pnpm.io/installation) to install pnpm globally on your machine.
 
-To have your global install of `platformatic` resolve to your local copy instead of the version installed via npm, you
-should use [`pnpm link`](https://pnpm.io/cli/link). This will add links so that you can use the `platformatic` and `plt`
-commands everywhere.
+3. **Setup pnpm**: Run the setup script to configure the local path and restart your termina.
 
-Run from the root folder of the project, execute the following.
+```sh
+pnpm setup 
+```
+
+4. **Install dependencies**: Install the dependencies for platformatic.
 
 ```sh
+pnpm install 
+```
+
+5. **Install Docker**: Install Docker using [Docker desktop](https://www.docker.com/products/docker-desktop) or [Colima](https://github.com/abiosoft/colima)
+
+To ensure your global installation of `platformatic` resolves to your local copy, use [`pnpm link`](https://pnpm.io/cli/link), this will add links so that you can use the `platformatic` and `plt`
+
+## Link Platformatic Globally 
+
+To ensure your global installation of `platformatic` resolves to your local copy, use [`pnpm link`](https://pnpm.io/cli/link), to add links so you can use `platformatic` and `plt` commands.
+
+From the root folder of the project, run:
+
+```sh 
 cd packages/cli
 pnpm link --global
```

Comment:
maybe we can add this as `pnpm run global-links` in the root project?

Note that we should do the same also for the `create-platformatic` project